docs/configuration/intra_node_data_path.md

```diff
@@ -124,11 +124,18 @@ CARET maps mechanically messages of `/pong` topic to messages of `/ping` which f
     `use_latest_message` and `callback_chain` do not cover all of use cases for CARET. We, CARET development team, continue improvements of intra-node data path definition.
 <prettier-ignore-end>
 
-### Python API
+## Define intra-node data path to the Architecture file/object
 
-Python API is not implemented so far. Python API support is planned in 2023.
+In order to measure latency in the inter-node data path, the Architecture file must be edited to match the entity of the application to be measured.
 
-### Architecture file editing
+There are two ways to define inter-node data path.
+
+1. Directly editing the Architecture file with an editor, etc
+2. Manipulating the Architecture object using the Python API
+
+Either way will produce the same results as long as it is defined correctly, so define the data path as you prefer.
+
+### Directly editing the file
```

Comment:
```suggestion
### Directly editing the Architecture file
```

Reply:
There is a concern that the names of the sub-sections displayed in the `Table of contents` will become too long.